nowcasting_dataset/data_sources/satellite/satellite_data_source.py

```diff
@@ -145,5 +145,7 @@ def open_sat_data(zarr_path: str, consolidated: bool) -> xr.DataArray:
     # To make it easier to align the satellite data with other data sources
     # (which are at 00, 05, ..., 55 minutes past the hour) we add 1 minute to
     # the time dimension.
+    # TODO Remove this as new Zarr already has the time fixed
```

Comment:
can you add a todo and an issue number ofr this?

Reply:
Yeah, I can